lib/index.d.ts

```diff
@@ -14,10 +14,20 @@ export * from "./common";
 export * from "./joint";
 export * from "./shape";
 export * from "../testbed";
+  
+export interface Manifold {
+  type: number;
+  localNormal: Vec2;
+  localPoint: Vec2;
+  points: Vec2[];
```

Comment:
I think the type of `points` should be `ManifoldPoint[]`.

Reply:
done.